lib/pull_preview/instance.rb

```diff
@@ -89,7 +89,7 @@ def public_dns
 
     def url
       scheme = (default_port == "443" ? "https" : "http")
-      "#{scheme}://#{public_dns}:#{default_port}"
+      "#{scheme}://#{public_dns}"
```

Comment:
default_port should be kept if different than a standard port, but yes can be removed if 443 or 80